frontend/components/ResourceInfoCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Grid, Paper, Sx, Text, rem } from '@mantine/core';
+import Link from 'next/link';
+import React from 'react';
+import { ResourceInfo } from '@/util/types/fhir';
+import { ArrowNarrowRight } from 'tabler-icons-react';
+
+export interface ResourceInfoCardProps {
+  resourceInfo: ResourceInfo;
+  selected?: boolean;
+}
+
+export default function PatientInfoCard({ resourceInfo, selected }: ResourceInfoCardProps) {
+  return (
+    <Paper
+      shadow="sm"
+      p="md"
+      sx={theme => {
+        const style: Sx = {
+          ':hover': {
+            cursor: 'pointer',
+            backgroundColor: theme.colors.gray[0]
+          },
+          borderRadius: 12,
+          border: `${rem(1)} solid ${theme.colors.gray[3]}`
+        };
+        if (selected) {
+          style.border = `${rem(1)} solid ${theme.colors.blue[3]}`;
+        }
+        return style;
+      }}
+    >
+      <Grid align="center">
+        <Grid.Col span={10}>
+          <div>
+            <Text size="md" fw={700}>
+              {resourceInfo.id}
+            </Text>
+          </div>
+          {selected && (
+            <div>
+              {Object.entries(resourceInfo).map(([key, val]) => {
+                return (
+                  val && (
+                    <Text size="sm" color="dimmed">
```

Comment:
These text nodes should have a unique key to avoid react rendering warnings:

<img width="823" alt="image" src="https://user-images.githubusercontent.com/16297930/236017302-7b7d68c6-4f10-4154-891a-7c64890c0865.png">